lib/pbench/server/api/resources/users_api.py

```diff
@@ -348,7 +327,6 @@ def get(self, username):
             try:
                 target_user = User.query(username=username)
                 response_object = {
-                    "message": "success",
                     "data": target_user.get_json(),
```

Comment:
Since we're only returning the User JSON, hiding it behind a `"data"` element seems obtuse; can we just return the object here?

Reply:
sure